fashion-messaging-app-api/server.js

```diff
@@ -139,6 +86,71 @@ app.get('/posts', async (req, res) => {
     res.status(500).json({ message: err.message });
   }
 });
+app.get('/photos', async (req, res) => {
+  const query = req.query.search || defaultQuery;
+
+  
+  try {
+    const response = await clientAPI.photos.search({ query, per_page: PHOTOS });
+    
+    let fetchPromises = response.photos.map(async photo => {
+      if (cache.has(photo.src.original)) {
+        return cache.get(photo.src.original);
+      
+      } else {
+        var myHeaders = new Headers();
+        myHeaders.append("x-api-key", "7008b5da328806c39a5561c3d51339c18dbb66dcd8bbfeebca80f0a96c44c332");           
+                     
+
+        var formdata = new FormData();
```

Comment:
nit: I know this was capitalized like this originally, but this should probably be in camel case (formData)! not a big deal. you did a good job with it everywhere else!

Reply:
> nit: I know this was capitalized like this originally, but this should probably be in camel case (formData)! not a big deal. you did a good job with it everywhere else!

Okay, got it!